### DISCUSSION TITLE
How to change voice from female to male?

### DISCUSSION THREAD

**Discussion by karti43** [pndurette/gTTS#361] [Q&A] [answered]
I can only use female voice in different accents, but i can't change voice to male can anyone help?

**Comment by pndurette** [comment, marked as answer]
Unfortunately, the way this library consumes the endpoints there's no way, as of this writing, to pick a voice for a given language.

**Reply by Mendpara-Mevil-4Born** [reply]
do you have any idea about other library that support changing the voice?
